front/src/StandardButton.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+/**
+ * Create a standard SRR Button
+ *
+ * Author: Tom Franken, Gijs van der Sanden
+ *
+ */
+
+const StandardButton = props => {
+  const { buttonText, onClick, enabled } = props;
+  return (
+    <button className={props.className} onClick={onClick} disabled={!enabled}>
+      {buttonText}
```

Comment:
you dont need to pass the button text as a prop.

Reply:
but it's nice.